### PR TITLE
[TC-7375] Add order type

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-7375

### Scope
This PR
- adds order type
- some improvements

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: buyer/issue/README.md [hunk 1]
```diff
@@ -85,25 +85,27 @@ Response status codes:
 
 ## Order
 
-* `companyId`: the optional Tradecloud company identifier. You only have to provide a companyId when your integration user account has multiple company permissions.
+* `companyId`: the optional Tradecloud company identifier. You only have to provide a companyId when your integration user account has multiple company authorization.
 * `supplierAccountNumber`: the supplier account number as known in your ERP system
+
+{% hint style="warning" %}
+The `supplierAccountNumber` should be set on forehand in the Tradecloud connection with your supplier. You can set the account code when inviting a new connection or in the connection overview in the portal.
```
Comment (Sjorstradecloud) [inline buyer/issue/README.md:92]: A user can only add the account number when the connection request is accepted by the other party. This probably will change when we rethink the "new company / new connection" process. Atm I link the companies and add account number as extra service, as all connection requests are related to companies that do not yet exist on TC One.

File: supplier/send-order-response/README.md [hunk 2]
```diff
@@ -113,17 +113,18 @@ Response status codes:
...
 
-* `companyId`: the optional Tradecloud company identifier. You only have to provide a companyId when your integration user account has multiple company permissions.
+* `companyId`: the optional Tradecloud company identifier. You only have to provide a companyId when your integration user account has multiple company authorization.
 * `buyerAccountNumber`: the buyer account number as known in your ERP system.
-* `purchaseOrderNumber`: the purchase order number as sent by the buyer.
-* `contact`: the supplier employee responsible for this order. You can either send his/her email or userName as known in your ERP system.
 
 {% hint style="warning" %}
-`buyerAccountNumber`, `contact.email` and `contact.userName` should be unique within your company and never change. Never renumber or re-use numbers or code's.
+The `buyerAccountNumber` should be set on forehand in the Tradecloud connection with your supplier. You can set the account code when inviting a new connection or in the connection overview in the portal.
```
Comment (Sjorstradecloud) [inline supplier/send-order-response/README.md:120]: Same as for supplierAccountNumber

File: supplier/send-order-response/README.md [hunk 3]
```diff
@@ -113,17 +113,18 @@ Response status codes:
...
 {% endhint %}
 
+* `purchaseOrderNumber`: the purchase order number as sent by the buyer.
+* `contact`: the supplier employee responsible for this order. You can either send his/her email or userName as known in your ERP system.
```
Comment (Sjorstradecloud) [inline supplier/send-order-response/README.md:124]: "userName as known in your ERP system." still not implemented.

File: buyer/issue/README.md (hunk 1 above)
Comment (marcmatt) [reply buyer/issue/README.md:92]: Changing to `The supplierAccountNumber should be set in the Tradecloud connection in the portal, after the connection request is accepted by the other party.`

File: supplier/send-order-response/README.md (hunk 2 above)
Comment (marcmatt) [reply supplier/send-order-response/README.md:120]: Changing to `The buyerAccountNumber should be set in the Tradecloud connection in the portal, after the connection request has been accepted by the other party.`

File: supplier/send-order-response/README.md (hunk 3 above)
Comment (marcmatt) [reply supplier/send-order-response/README.md:124]: Removed everything with `userName`

**Review by Sjorstradecloud — APPROVED** [review]

**Pull request merged**